lib/api/include/irods/getHostForGet.h

```diff
@@ -17,4 +17,9 @@ extern "C"
 #endif
 int rcGetHostForGet( rcComm_t *conn, dataObjInp_t *dataObjInp, char **outHost );
 
+#ifdef __cplusplus
+extern "C"
+#endif
+int rc_get_rescinfo_for_get( rcComm_t *conn, dataObjInp_t *dataObjInp, char **outHost );
```

Comment:
Should this be in its own header file? Same for the "rs" and "put" variants.

Reply:
I've moved things into their own files as much as possible.